rmgpy/chemkin.pyx

```diff
@@ -1917,22 +1917,22 @@ def mark_duplicate_reaction(test_reaction, reaction_list):
             # duplicates of one another.
             # RHW question: why can't TemplateReaction be duplicate of LibraryReaction, in Chemkin terms? I guess it shouldn't happen in RMG.
             continue
-        same_dir_match = (reaction1.reactants == reaction2.reactants and reaction1.products == reaction2.products)
-        opposite_dir_match = (reaction1.products == reaction2.reactants and reaction1.reactants == reaction2.products)
+        same_dir_match = (sorted(reaction1.reactants) == sorted(reaction2.reactants) and sorted(reaction1.products) == sorted(reaction2.products))
+        opposite_dir_match = (sorted(reaction1.products) == sorted(reaction2.reactants) and sorted(reaction1.reactants) == sorted(reaction2.products))
```

Comment:
There may need to be an edit here to catch errors like 
`InputFileError thrown by Kinetics::checkDuplicates:
Undeclared duplicate reactions detected:
Reaction ####: A <=> A
Reaction ####: 2 A <=> 2 B`

that results when you use either the ck2cti or ck2yaml.